src/librustc/traits/on_unimplemented.rs

```diff
@@ -85,24 +87,33 @@ impl<'tcx> OnUnimplementedDirective {
         let mut message = None;
         let mut label = None;
         let mut note = None;
+        let mut enclosing_scope = None;
         let mut subcommands = vec![];
+
+        let parse_value = |value_str| {
+                OnUnimplementedFormatString::try_parse(tcx, trait_def_id, value_str, span)
+                .map(Some)
+            };
```

Comment:
minor nit: could you adjust the indentation here so that it's more like this:

```rust
let parse_value = |value_str| {
    OnUnimplementedFormatString::try_parse(tcx, trait_def_id, value_str, span)
        .map(Some)
};
```

This is mostly personal preference, but this is how I've seen code like this indented in the compiler normally.